charts/tidb-lightning/templates/persistentvolumeclaim.yaml

```diff
@@ -18,3 +18,23 @@ spec:
     requests:
       storage: {{ .Values.dataSource.remote.storage }}
 {{ end -}}
+---
+{{ if eq .Values.backend "local" }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ include "tidb-lightning.name" . }}-sorted-kv
+  labels:
+    app.kubernetes.io/name: {{ include "tidb-lightning.name" . }}-sorted-kv
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Values.targetTidbCluster.name }}
+    app.kubernetes.io/component: tidb-lightning
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+"  "_" }}
+spec:
+  storageClassName: {{ .Values.sortedKV.storageClassName }}
```

Comment:
If we specify `.Values.dataSource.local`, then we will set `.spec.nodeName`. but this can't work if the storage-class has `waitForFirstConsumer`. ref https://github.com/openebs/openebs/issues/2915 and https://github.com/kubernetes/kubernetes/issues/89953.

Should we need to:
- just document this behavior
- or update the chart to use `nodeSelector` or `nodeAffinity` rather than `nodeName`

WDYT @DanielZhangQD

Reply:
@csuzhangxc If `.Values.dataSource.local` is used, we can also use that hostPath to save the sst files instead of using PV.

Reply:
but for Lightning, It may be I/O bound. If we use the source hostPath to save sst files. It may become slower.

BTW, I'm also thinking about where can we store checkpoint (#3620) for the same reason. for Lightning, we often suggest the user to store the checkpoint neither in the source disk nor the target TiDB cluster.

Reply:
`or update the chart to use nodeSelector or nodeAffinity rather than nodeName` this is good.

Reply:
updated to `nodeSelector` in https://github.com/pingcap/tidb-operator/pull/3644/commits/2f33eb47006dd5be820eabf1ad4260322252ad21.